src/stages/evaluate.ts

```diff
@@ -273,6 +277,7 @@ function getAbilitiesCompleted<T, G>(getState: () => WolfState, abilities: Abili
  */
 function isAbilityCompletedByFilledSlotsOnCurrentTurn<T, G>(
   getState: () => WolfState,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1